block-storage/src/main/scala/coop/rchain/blockstorage/dag/BlockDagRepresentationSyntax.scala

```diff
@@ -100,4 +113,19 @@ final class BlockDagRepresentationOps[F[_]: Sync](
       ib <- dag.invalidBlocks
       r  = ib.map(block => (block.blockHash, block.sender)).toMap
     } yield r
+
+  def getCasperJustificationsUnsafe(
+      blockHash: BlockHash
+  ): F[Set[Casper.Justification[BlockHash, Validator]]] =
+    lookupUnsafe(blockHash).map(
+      _.justifications.map {
+        case Justification(validator, latestBlockHash) =>
+          Casper.Justification(latestBlockHash, validator)
+      }.toSet
+    )
+
+  def toCasperJustificationUnsafe(
+      blockHash: BlockHash
+  ): F[Casper.Justification[BlockHash, Validator]] =
+    lookupUnsafe(blockHash).map(m => Casper.Justification(m.blockHash, m.sender))
```

Comment:
`getCasperJustificationsUnsafe` and `toCasperJustificationUnsafe` should be moved to BlockDagRepresentationSyntax in casper project.